test/index.test.ts

```diff
@@ -59,10 +59,21 @@ afterAll(async () => {
 const commands = {
   vanilla: [projectName, ENTER, ENTER, ENTER, ENTER, ENTER],
 };
+const flags = {
+  vanilla: [projectName, '-t', 'vanilla', '-l', 'id', '--js', '--use-npm'],
+};
```

Comment:
Thank you for adding the test!
`yarn test` seems to occur error
I think so we need to change `--use-npm` flag to `--use-yarn`.

Reply:
Thank you for providing a detailed comment, it's now fixed!